example/pom.xml

```diff
@@ -74,7 +74,17 @@ Also contains a sample Docker Compose file (see &quot;docker&quot; folder) for s
       <resource>
         <directory>${project.basedir}/docker</directory>
         <filtering>true</filtering>
-        <targetPath>${project.build.directory}/hono</targetPath>
+        <targetPath>${project.build.directory}/hono/example/docker</targetPath>
+      </resource>
+      <resource>
+        <directory>${project.basedir}/openshift</directory>
```

Comment:
move all deploy *src* scripts under a common base folder so that we need to define one resource here, e.g
keeping sources in
src/main/deploy/docker
src/main/deploy/kubernetes
src/main/deploy/openshift
and then just define
```
<resource>
  <directory>${project.basedir}/src/main/deploy</directory>
  <filtering>true</true>
  <targetPath>${project.build.directory}/deploy</targetPath>
</resource>
```
?